README.md

```diff
@@ -91,7 +91,10 @@ In `package.json`, inside `jest` section, the `transform` should be like this:
 ```
 
 ## Options
-By default this package will try to locate `tsconfig.json` and use its compiler options for your `.ts` and `.tsx` files.  
+By default this package will try to locate `tsconfig.json` and use its compiler options for your `.ts` and `.tsx` files.
+
+Additionaly, `module` property will be overwritten to `commonjs` since is what Jest expects.
```

Comment:
minor nit

'since is what Jest expects' -> 'since that is the format Jest expects'